lib/src/rocket.rs

```diff
@@ -604,6 +604,25 @@ impl Rocket {
         self
     }
 
+    /// Get the managed state of type T, if it exists.
```

Comment:
Let's say...

"Returns `Some` of the managed state value for the type `T` if it exists. Otherwise, returns `None`."